modules/server_processes_manager/php/mriuploadserverprocess.class.inc

```diff
@@ -105,6 +99,26 @@ class MriUploadServerProcess extends AbstractServerProcess
             throw $exception;
         }
 
+        try {
+            $this->_prodFile = $config->getSetting('MriConfigFile');
```

Comment:
There shouldn't never be a try/catch around getSetting. If the setting isn't present and the code depends on it, the server is misconfigured and it should die loudly so that the server can be fixed.

Reply:
@driusan I simply reproduces the code for the other ConfigSettings (see line 98). Should I change this for all or just the ones I added. Let me know. Thanks!!

Reply:
Those ones shouldn't have gotten in either.. it's probably best to fix them both so that people don't try copying it later.